### DISCUSSION TITLE
How to configure infrastructure as code on Supabase ?

### DISCUSSION THREAD

**Discussion by glassmonkey** [supabase/supabase#3078] [Questions] [answered]
Supabase is a very attractive product for me.
I would like to use IInfrastructure as Code to manage and leverage the Supabase configuration.
Do you have any plans to offer a Terraform provider?

**Comment by glassmonkey** [comment]
The only way to achieve IInfrastructure as Code is to handle the CLI?

**Comment by sweatybridge** [comment, marked as answer]
We have released an official [terraform provider](https://github.com/supabase/terraform-provider-supabase) for managing projects and their configs. If you run into any difficulties, feel free to file an issue or post to the [main discussion](https://github.com/orgs/supabase/discussions/5166).
